src/Extensions/resendpost.py

```diff
@@ -0,0 +1,43 @@
+from discord.ext import tasks, commands
+import discord
+import base64
+
+
+def setup(bot):
+    bot.add_cog(ResendPost(bot, bot.post_queue))
+
+
+class ResendPost(commands.Cog):
+    def __init__(self, bot, post_queue):
+        self.bot = bot
+        self.aviso_channels = None
+        self.resend_post.start()
+        self.post_queue = post_queue
+
+    def cog_unload(self):
+        self.resend_post.cancel()
+
+    @tasks.loop(seconds=5.0)
+    async def resend_post(self):
+        """check for post in the queue each 5 seconds
+        """
+        if not self.post_queue.empty():
+            post = self.post_queue.get()
+            image = None
+            if post['photo']:
+                image_str = str(post['photo']).encode('ascii')
+                image_byte = base64.b64decode(image_str)
+                image = open('anuncio.png', 'wb+')
+                image.write(image_byte)
+                image.close()
```

Comment:
```suggestion
                with open('anuncio.png', 'wb+') as image:  
                    image.write(image_byte)
```
Es nitpick, pero en este caso creo que es mejor usar with si solo vas a hacer un write (el with cierra el archivo cuando acaba)